azurerm/internal/services/storage/parse/storage_filesystem_ace.go

```diff
@@ -0,0 +1,69 @@
+package parse
```

Comment:
_minor_ we're using the `parse` folder for Resource ID Parsers rather than for Schema -> Object parsing - could we move this back up a level